docs/core/diagnostics/runtime-events.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Runtime Events"
+description: Review events emitted by CoreCLR runtime.
```

Comment:
Need more keywords here and things to help search. Words like .NET, diagnostic, etc.

Titles are sentence cased

```suggestion
title: "Runtime events"
description: Review events emitted by CoreCLR runtime.
```